plugin/pkg/admission/serviceplan/defaultserviceplan/admission.go

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
This admission controller's behavior as it stands would be completely undocumented. Could we get a short message somewhere in our docs that mentions it?

I guess the most appropriate place would be: https://github.com/kubernetes-incubator/service-catalog/blob/master/docs/design.md#creating-a-service-instance . It's rough around the edges for now, but at least having a mention would remind us to include it in a rewrite should we do so later.

Reply:
:cry: